src/api_routes_http.rs

```diff
@@ -347,6 +348,12 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/community", web::post().to(route_post::<PurgeCommunity>))
           .route("/post", web::post().to(route_post::<PurgePost>))
           .route("/comment", web::post().to(route_post::<PurgeComment>)),
+      )
+      .service(
+        web::scope("/custom_emoji")
+          .route("", web::post().to(route_post_crud::<CreateCustomEmoji>))
+          .route("", web::put().to(route_post_crud::<EditCustomEmoji>))
+          .route("", web::delete().to(route_post_crud::<DeleteCustomEmoji>)),
```

Comment:
This is the very first instance of `web::delete()`, all of the others do a post at `/delete`. Keep this consistent with the others, and if we do switch to delete, it should be all of them uniformly.